Dockerfile

```
@@ -1,26 +1,29 @@
 FROM python:3.9-slim
 
+# Set working directory
 WORKDIR /app
-RUN pip install --upgrade pip
-COPY requirements.txt requirements.txt
-RUN pip3 install --no-cache-dir -r requirements.txt
+
+# Install system dependencies
 RUN apt-get update && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)